app/uk/gov/hmrc/uknwauthcheckerapi/models/constants/MinMaxValues.scala

```diff
@@ -14,9 +14,9 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.uknwauthcheckerapi.utils
+package uk.gov.hmrc.uknwauthcheckerapi.models.constants
 
-object JsonErrors {
-  val expectedJsObject = "error.expected.jsobject"
-  val pathMissing      = "error.path.missing"
+object MinMaxValues {
+  val maxEoriCount: Int = 3000
```

Comment:
Just curious and for my understanding, would it be a good idea to maybe have these as config values in AppConfig, or does it not really matter as I'm assuming they will probably never change?

Reply:
These are hardcoded in PDS, so i don't believe we need to make them configurable at this time. If there's a CR to increase or reduce the limits, we can make this change.